Update docker-compose examples, tweak docs related to Grafana 7.4 (#873)
grafana/tempo: docs/tempo/website/configuration/querying.md

The way Grafana queries Tempo changed from 7.4.x to 7.5.x. This document aims to explain the difference between the two
and help you set up your datasources appropriately.

## Grafana 7.5.x and higher (easy mode)

> Grafana 7.5.x is able to query Tempo directly. Point the Grafana datasource at your Tempo query frontend (or single

Reviewer: ```suggestion
Grafana 7.5.x and higher can query Tempo directly. Point the Grafana data source at your Tempo query frontend (or single
```

> binary) and enter the url: `http://<tempo hostname>:<http port number>`. For most of [our examples](https://github.com/grafana/tempo/tree/main/example/docker-compose) the following works.

Reviewer: ```suggestion
binary) and enter the URL: `http://<tempo hostname>:<http port number>`. For most of [our examples](https://github.com/grafana/tempo/tree/main/example/docker-compose) the following works.
```
